recipe/meta.yaml

```diff
@@ -48,7 +48,6 @@ requirements:
     - libpng
     - ffmpeg 4.1.*                   # [not win]
     - qt                             # [not osx]
-    - liblapacke
```

Comment:
My initial thought is to keep liblapcke as a dependency even if we don't use it.

This will help the solver avoid selecting this specific version of opencv to avoid a lapacke dependency while we add lalack back in.